desktop/core/src/desktop/js/reactComponents/FileChooser/PathBrowser/Breadcrumb/Breadcrumb.tsx

```diff
@@ -26,9 +28,18 @@ interface BreadcrumbProps {
 }
 
 const Breadcrumb: React.FC<BreadcrumbProps> = ({ label, url, onFilepathChange }) => {
+  const onClick = () => {
+    onFilepathChange(url);
+  };
+
+  const minWidth = '' + (label.length < 10 ? label.length : 10) + 'ch';
```

Comment:
Just a FYI. The 'ch' unit is based on the '0' character so unless you use monospace a label "wwwww" will not have enough space in "5ch" since a 'w' is wider than a '0'.